### DISCUSSION TITLE
Stress tensor for scratch test

### DISCUSSION THREAD

**Discussion by Mustafa-Rifat** [mtex-toolbox/mtex#635] [Announcements]
Hi,

I was going through the documentation of schmid factor calculation. I want to see the slip trace of the grains. I have performed a scratch test on my sample. The ebsd map is shown below with the x and y direction. I was wondering what my stress tensor should be for this. Scratch is not uniaxial as it both indents and moves forward. So there should be a z component of the stress tensor along with a y component, according to the image attached. 
My question is: should the stress tensor be:
`sigma = stressTensor(diag([0 1 1]));`
`sigma = stressTensor(diag([0 1 -1]));`

<img width="343" alt="ebsd" src="https://user-images.githubusercontent.com/60717121/102557091-6645f700-4098-11eb-900e-ba6e733221bb.png">

**Comment by kilir** [comment]
Hi,
from my point of view, as soon as things move, the stress tensor for every point is a function of the material response to being displaced/deformed, e.g. by the scratching thing you use. For a constant displacement rate, it will also depend on the orientation of your grain.
Cheers,
Rüdiger

**Reply by Mustafa-Rifat** [reply]
So it won't be as simple as:
`sigma = stressTensor(diag([0 1 1]));`
or `sigma = stressTensor(diag([0 1 -1]));`?
How do I get the stress tensor values? Finite Element Analysis?
